include/bluetooth/services/fast_pair.h

```diff
@@ -77,13 +80,35 @@ enum bt_fast_pair_battery_comp {
 	BT_FAST_PAIR_BATTERY_COMP_COUNT
 };
 
+/** @brief Fast Pair discoverable advertising info. Currently empty. */
+struct bt_fast_pair_disc_adv_info {};
+
+/** @brief Fast Pair not discoverable advertising info. */
+struct bt_fast_pair_not_disc_adv_info {
+	/** Advertising type. According to Fast Pair specification, when no Account Key has been
+	 *  saved on the device, this field has no effect on the advertising data.
+	 */
+	enum bt_fast_pair_not_disc_adv_type type;
+
+	/** Fast Pair advertising battery mode. */
+	enum bt_fast_pair_adv_battery_mode battery_mode;
```

Comment:
Maybe we could also state that battery values are set using a separate API (`bt_fast_pair_battery_set`)